packages/unlock-js/openapi.yml

```diff
@@ -1153,10 +1125,10 @@ paths:
 
   /v2/api/ticket/{network}/lock/{lockAddress}/key/{keyId}/check:
     put:
-      operationId: checkTicket
```

Comment:
Let's not change this because that is breaking change for other people who are using the API.
Also. I think you can add a `deprecated` field here.